content/v2/zones/ns-records.markdown

```diff
@@ -18,7 +18,7 @@ excerpt: This page documents the DNSimple domain name servers API v2.
 
     PUT /:account/zones/:zone/ns_records
 
-Update the NS records for a zone in the account.
+Update the NS records for a zone in a hosted domain in the account. If vanity name servers are enabled for the domain, plesae ensure that they are included in the name server name parameters or referenced name server sets, for the vanity name server configuration to stay effective.
```

Comment:
```suggestion
Update the NS records for a zone in a hosted domain in the account. If vanity name servers are enabled for the domain, please ensure that they are included in the name server name parameters or referenced name server sets, for the vanity name server configuration to stay effective.
```

Reply:
WDYT wrapping this in `<info>` or `<warning>` tags for emphasis?

```
Update the NS records for a zone in a hosted domain in the account

<info>
If vanity name servers are enabled for the domain, please ensure...
</info>
```

Reply:
Great idea! Wrapped the info in `<info>` tags 👍